FAQ.md

```diff
@@ -0,0 +1,63 @@
+---
+status: draft
+---
+
+# Frequently Asked Questions (FAQs)
+
+## Why create a new foundation?
+
+There are already several foundations which host open source projects,
+and each has its own focus and requirements.
+The CF is a home for self-governing, code-centric open source projects.
+Each open source project has its own unique needs and goals,
+and not every foundation is the right fit for every project.
+
+There are many critical application frameworks and libraries within the open source ecosystem that are not currently part of a foundation.
+The CF is a new option for projects to consider when looking for a foundation to call home,
+helping to strengthen the open source ecosystem as a whole by ensuring more projects are sustainable in the long term.
+
+Information covering benefits of the CF are in the [README](README.md):
+
+- [Core Distinctions of the CF](README.md#core-distinctions-of-cf)
+- [CF Guiding Principles](README.md#our-guiding-principles)
+- [Focus and Vision](README.md#cfs-focus-and-vision)
+- [Community and Sponsorship](README.md#community-and-sponsorship)
+
+## Why not the Apache Software Foundation?
+
+The Apache Software Foundation (ASF) has a long standing tradition of being a home for open source projects of all types,
+and continues to be a great home for many.
+As part of a project joining the ASF, it is required to run on the infrastructure that is provided for projects, including mailing lists,
+and adhere to its processes.
+
+For projects with well established infrastructure and processes, it's preferable for these to remain unchanged when joining a foundation.
```

Comment:
```suggestion
For projects with well established infrastructure and processes, it might be preferable for these to remain unchanged when joining a foundation.
```